src/App/PropertyLinks.cpp

```diff
@@ -3598,6 +3791,10 @@ void PropertyXLink::Save (Base::Writer &writer) const {
             << "<XLink file=\"" << encodeAttribute(path)
             << "\" stamp=\"" << (docInfo&&docInfo->pcDoc?docInfo->pcDoc->LastModifiedDate.getValue():"")
             << "\" name=\"" << objectName;
+
+//        if (this->resolveMode != PathResolveMode::Dynamic) {

```

Comment:
As usual - commented code should either not be commited or have comment explaining why it is left.

Reply:
An artifact of the series of stacked PRs - this was actually fixed in 15501 but had not yet been rebased.  There will be a lot of these in the remaining PRs in the series.